packages/flutter/lib/src/cupertino/tab_scaffold.dart

```diff
@@ -120,6 +201,12 @@ class CupertinoTabScaffold extends StatefulWidget {
   /// Must not be null.
   final CupertinoTabBar tabBar;
 
+  /// Controls the current selected tab item of the [tabBar], as well as the
+  /// the currently focused tab from the [tabBuilder].
+  ///
+  /// Defaults to null.
```

Comment:
Mention that changing the controller will update the scaffold and the tab bar to the new controller's index value.